assignments/week-1/heart.k

```diff
@@ -0,0 +1,55 @@
+function getBeepers() {
+	while (beepersPresent()) {
+		pickBeeper();
+	}
+
+}
+
+function putBeepers() {
+	while (beepersInBag()) {
+		putBeeper();
+		move();
+	}
+}
+
+function turnRight() {
+	turnLeft();
+	turnLeft();
+	turnLeft();
+}
+
+function turnAround(){
+	turnLeft();
+	turnLeft();
+	while (frontIsClear()) {
+		move();
+	}
+	turnRight();
+	move();
+	turnRight();
+}
+
+function repeatRows() {
+	repeat(11) {
+		while (frontIsClear()) {
+			move();
+			getBeepers();
+			putBeepers();	
```

Comment:
It's somewhat confusing to have a move command in this loop and to also have `putBeepers` itself move - seems like the fn responsibilities are conflated.